src/components/CouponCard/CouponCard.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+import { Coupon } from '../../types/coupon';
+import CheckBox from '../CheckBox/CheckBox';
+import Title from '../Title/Title';
+import { useRecoilState } from 'recoil';
+import { couponDetailState } from '../../recoil/atoms';
+import { formatAvailableTime, formatDate } from '../../utils/Time';
+import useCouponApplicable from '../../hooks/useCouponApplicable';
+
+const CardContainer = styled.li<{ $disabled: boolean }>`
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 0 0 0;
+  gap: 1rem;
+  border-color: rgba(0, 0, 0, 0.1);
+  border-width: 0.5px 0 0 0;
+  border-style: solid;
+  color: ${(props) => (props.$disabled ? 'rgba(0, 0, 0, 0.1)' : '')};
+`;
+
+const CardHeader = styled.div`
+  display: flex;
+  gap: 5px;
+`;
+
+const CardDetail = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+`;
+interface CouponCardProps {
+  coupon: Coupon;
+}
+function CouponCard({ coupon }: CouponCardProps) {
+  const [couponDetail, setCouponDetail] = useRecoilState(
+    couponDetailState(coupon.id),
+  );
+  const disabled = !useCouponApplicable(coupon, couponDetail);
```

Comment:
보통 customHook을 사용할 때, 한번의 추상화 개념이 있어서 그런걸 수도 있는데, 변수로 한번 빼내고 값을 사용하는경우가 많더라구요! 그리고, 사용처에서도 어떤 경우에 disable되는구나를 명확히 할 수도 있겠죠
(`disabled={disabled}`와 `disabled={isCouponApplicable}`의 차이)
물론 지금 처럼 직관적으로 알 수 있는 상황에선 CouponApplicable을 반환하는 거겠구나 싶긴 하겠지만 혹시나 해서 comment 남겨요!
```suggestion
  const isCouponApplicable = useCouponApplicable(coupon, couponDetail);
  
  // 사용처
  disabled={!isCouponApplicable}
```